cranecloud/docs/guides/deployHtmlApp.md

```diff
@@ -0,0 +1,89 @@
+Getting an application running on Crane Cloud is essentially working out how to package it as a deployable image. Once packaged it can be deployed to the Crane Cloud platform.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud. See [demo here](https://my-html-app-578060fa-3901-4464-8f3d.cranecloud.io/)
+
+## Prerequisites
+
+Before we begin, we assume that you already have the following:
+
+1. Fork [sample-html-app](https://github.com/crane-cloud-Apps/sample-html-app) on Github.
+
+2. Docker installed on your machine. If you haven't got Docker installed, you can use [this resource](https://docs.docker.com/get-docker/) to get you up and running.
+
+3. Docker Hub account. If you need to create one first, [click here.](https://hub.docker.com/signup)
+
+## Create a Dockerfile for your app
+
+In our application's directory, we shall create a `Dockerfile` and in it, let us write the following commands
+
+```js
```

Comment:
`js` on line 21 here seems redundant and confusing to the reader

Reply:
thank you.

Reply:
I intentionally put the "js" to give the Dockerfile color while in the IDE, and has not effect on the UI docs.